app/presenters/detailed_guide_presenter.rb

```diff
@@ -43,6 +43,33 @@ def related_mainstream
     links("related_mainstream")
   end
 
+  def national_applicability
+    content_item["details"]["national_applicability"]
+  end
+
+  def applies_to
+    return nil if !national_applicability
+    all_nations = national_applicability.values
+    applicable_nations = all_nations.select { |n| n["applicable"] }
+    inapplicable_nations = all_nations - applicable_nations
+
+    applies_to = applicable_nations.map { |n| n["label"] }.to_sentence
+
+    if inapplicable_nations.any?
+      nations_with_alt_urls = inapplicable_nations.select { |n| n["alternative_url"] && !n["alternative_url"].empty? }
+      if nations_with_alt_urls.any?
+        additional_guidance_links = nations_with_alt_urls
+          .map { |n| "<a class=\"country\" rel=\"external\" href=\"#{n['alternative_url']}\">#{n['label']}</a>" }
+          .to_sentence
+        # TODO: Translation for `see detailed guidance for` below?
+        additional_guidance = " (see detailed guidance for #{additional_guidance_links})"
```

Comment:
This "detailed guidance" bit won't apply to all formats that need to use this pattern.